msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -801,6 +801,28 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
     |`MicrophoneStream.Read.User`| Allows the app to read user's microphone stream.|
     |`MeetingParticipantReaction.Read.User`| Allows the app to read user's reactions while participating in a meeting.|
 
+## extensions
+
+**Optional**—array
+
+Contains objects that define the set of extensions for the app. Used to specify Outlook Add-ins within a unified manifest for simplified distribution and acquisition within the Microsoft 365 ecosystem.
```

Comment:
Can we simplify this to:

"Used to specify Outlook Add-ins within an app manifest for simplified distribution and acquisition within the Microsoft 365 ecosystem."


Using the term "unified" manifest begs the natural question what is a non-unified manifest, and this might only cause confusion.

Reply:
Text should be updated in latest commit.